Use /proc/self/fd/1 instead of /dev/stdout, so we can use
`kmod static-nodes` before udev is running.

--- a/tools/static-nodes.c
+++ b/tools/static-nodes.c
@@ -156,7 +156,7 @@
 {
 	struct utsname kernel;
 	char modules[PATH_MAX], buf[4096];
-	const char *output = "/dev/stdout";
+	const char *output = "/proc/self/fd/1";
 	FILE *in = NULL, *out = NULL;
 	const struct static_nodes_format *format = &static_nodes_format_human;
 	int r, ret = EXIT_SUCCESS;
